helm/templates/_helpers.tpl

```diff
@@ -46,9 +46,9 @@ capabilities:
     - ALL
 privileged: false
 readOnlyRootFilesystem: true
-runAsGroup: 2000
```

Comment:
Why 1000:2000 in Dockerfiles, but not in config?

Reply:
trivy recommends >10000. Could try to adjust the Dockerfile :shrug:

Reply:
k8s sec page does 1k:2k:3k https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
Would be interested in why trivy wants it higher

Reply:
Guess it's collision blabla, but our image isn't even clone to using the uids and guids; don't like changing existing code for "tool said pls change" reasons, but not enough to not approve